qt5connectivity: Don't build scanner example without QtQuick
Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>

From 1454664752505f5870e4173d15362bfdfe4148e7 Mon Sep 17 00:00:00 2001
From: Alex Blasche <alexander.blasche@digia.com>
Date: Wed, 15 Jan 2014 12:11:31 +0100
Subject: [PATCH] Don't build scanner example without QtQuick

Task-number: QTBUG-35711
Change-Id: I0406fb5edd8fb2a90046308853485eaaa4311a26
Reviewed-by: Friedemann Kleint <Friedemann.Kleint@digia.com>
Reviewed-by: Fabian Bumberger <fbumberger@rim.com>
Reviewed-by: Alex Blasche <alexander.blasche@digia.com>
---
 examples/bluetooth/bluetooth.pro |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/examples/bluetooth/bluetooth.pro b/examples/bluetooth/bluetooth.pro
index e3ef769..549bb7c 100644
--- a/examples/bluetooth/bluetooth.pro
+++ b/examples/bluetooth/bluetooth.pro
@@ -6,4 +6,4 @@ qtHaveModule(widgets) {
                bttennis
 }
 
-SUBDIRS += scanner
+qtHaveModule(quick): SUBDIRS += scanner
-- 
1.7.1

